code/evaluate/command.py

```diff
@@ -175,7 +175,7 @@ def command(shell: DTShell, args, **kwargs):
             exit(1)
 
         # make sure we have the credentials to push to this registry
-        shell_cfg: ShellConfig = read_shell_config()
+        shell_cfg: ShellConfig = read_shell_config(shell)
```

Comment:
This might break on the Duckietown Shell v5 and lower as that function did not have an argument before. We should try-catch for TypeError instead.